_posts/2020-01-07-quarkus-eclipse-microprofile-3-2-compatible.adoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Quarkus is Eclipse MicroProfile 3.2 compatible!
+date: 2020-01-07
+tags: quarkus microprofile
+author: kenfinnigan
+---
+
+With the recent announcement that Quarkus 1.1 is now available,
+and after a tremendous effort from all the contributors and community,
+we’re also excited to announce that Quarkus 1.1 is Eclipse MicroProfile 3.2 compatible!
+https://projects.eclipse.org/projects/technology.microprofile/releases/microprofile-3.2[MicroProfile 3.2] was released this week,
+bringing updates to the Metrics and Health specifications,
+but what is MicroProfile and why do I care?
+https://microprofile.io/[MicroProfile] creates specifications for developing microservices with Enterprise Java,
+making compatibility with MicroProfile a key feature of Quarkus.
+
+Over the last couple of months,
+the team has worked tirelessly to add the various MicroProfile specification TCKs to the Quarkus build and hunt down the failures they raised.
+With fixes not just in Quarkus,
+but also fixes for dependent libraries such as https://smallrye.io/[SmallRye] and https://resteasy.github.io/[RESTEasy].
+
+With Quarkus we’re seeking the best developer experience for cloud-native development,
+and sometimes that means making tough decisions.
+With ArC we sought to create an opinionated CDI implementation optimized for the cloud and containers, and in particular, Kubernetes.
+ArC isn’t a full CDI implementation because it’s engineered to take full advantage of the build time optimization within Quarkus,
+enabling supersonic runtime startup and subatomic memory utilization.
+The slight drawback is that not all CDI features are available,
+but features that developers generally use on a day-to-day basis are present.
+Check out https://quarkus.io/guides/cdi-reference for the details.
```

Comment:
Other hyperlinks contain a label (in square brackets), perhaps this one could too for consistency?